server/services/BpmnToSsotParsing/BpmnToSsotParsing.jsx

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable no-underscore-dangle */
+
+/**
+ * @category Server
+ * @module
+ */
+
+/**
+ * @description Creates a base element of the single source of truth
+ * @returns {object}  Base element of the single source of truth
+ */
+const createBaseElement = (id) => {
+    const baseElement = {
+        "type": "",
+        "name": "",
+        "id": id,
+        "predecessorIds": [],
+        "successorIds": []
+    }
+    return baseElement
+};
+
+/**
+ * @description Creates the metadata object of the single source of truth
+ * @returns {object}  Metadata object of the single source of truth
+ */
+const createMetadataObj = () => {
+    const metadataObj = {
+        "robotId": "exampleRobot",
+        "starterId": "exampleID"
+    }
+    return metadataObj
+};
+
```

Comment:
I would appreciate it, if maybe the baseElement and metadataObj would be in a separate file. This could allow for more ease when having to change something and keeps objects separated from functionality to some extend.
But I am open to discussion, as I am only getting a sneak peek into the thoughts behind the functionality